examples/builder_pattern/main.rs

```diff
@@ -1,6 +1,13 @@
 use gtk::prelude::*;
 use gtk::{Align, Application, ApplicationWindowBuilder, ButtonBuilder};
 
+fn main() {
+    let application =
+        Application::new(Some("com.github.gtk-rs.examples.basic"), Default::default());
```

Comment:
```suggestion
        Application::new(Some("com.github.gtk-rs.examples.builder_pattern"), Default::default());
```
it's a bit unrelated, but let's use a different app-id per example if possible